app/controllers/species_lists/create_and_edit.rb

```diff
@@ -87,6 +90,25 @@ def update
     process_species_list(:update)
   end
 
+  # Used by show_species_list.
+  def make_report
```

Comment:
Restores action accidentally removed in transition from `master` to `bootstrap4`.